pre-check/v1.x/check.sh

```diff
@@ -134,8 +134,27 @@ check_nodes()
         rm -f $node_ready_state
     fi
 
+    # log to customer that cluster has witness nodes
+    witness=$(echo "$nodes" | yq '.items[] | select(.metadata.labels."node-role.harvesterhci.io/witness" == "true")' )
+    if [ -n "$witness"]; then
+        log_info "There are witness nodes in your cluster."
+        witness_name=$(echo "$witness" | yq .metadata.name)
```

Comment:
could you also ensure that there is at most 1 witness node, in case unluckily more nodes are having such label